### PR TITLE
Persistent Reverse Shell - Telegram Based

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

File: payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Persistent Reverse Shell - Telegram Based
+
+A script used to configure a persistent reverse shell on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md:3]: through*

File: payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Category**: Execution
+
+## Dependencies
+
+* Internet Connection
+
+## Description
+
+A script used to configure a persistent reverse shell on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md:13]: through*

File: payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md (hunk 1 above)
Comment (aleff-github) [reply payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md]: ad27b19b791dfda789c0791b5b28a3405934277d

File: payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md (hunk 2 above)
Comment (aleff-github) [reply payloads/library/execution/Persistent_Reverse_Shell-Telegram_Based/README.md]: Solved > ad27b19b791dfda789c0791b5b28a3405934277d

**Pull request merged**